### ISSUE TITLE
Does model support different camera model ?

### ISSUE THREAD

**Issue by lt-xiang** [zju3dv/DetectorFreeSfM#22]
Hi, I have noticed that camera model exported from "colmap_refined" folder is ”SIMPLE_RADIAL“. May I ask whether code support different camera model (such as SIMPLE_PINHOLE, PINHOLE) ?

![image](https://github.com/zju3dv/DetectorFreeSfM/assets/27660078/83dd63e4-1c39-49c4-a6d4-0c2206c68982)

**Comment by hxy-123** [comment]
Yes, you can set `neuralsfm.colmap_cfg.ImageReader_camera_model='SIMPLE_PINHOLE'`, or `PINHOLE`.
